adapters/risc0/src/host.rs

```diff
@@ -9,12 +9,14 @@ use sov_rollup_interface::zk::{Zkvm, ZkvmHost};
 
 use crate::Risc0MethodId;
 
+/// Allows to run prover code and send data to ZkVM
 pub struct Risc0Host<'a> {
     env: RefCell<ExecutorEnvBuilder<'a>>,
     elf: &'a [u8],
 }
 
 impl<'a> Risc0Host<'a> {
+    /// Create a new host fo a given reference to an ELF file
```

Comment:
nit: It's not obvious what it means that a host is "for a given reference to an ELF file". Maybe "Create a host to prove the provided ELF file?"